zcash_primitives/src/sapling/test_vectors/signatures.rs

```diff
@@ -0,0 +1,476 @@
+pub(crate) struct TestVector {
+    pub(crate) sk: [u8; 32],
+    pub(crate) vk: [u8; 32],
+    pub(crate) alpha: [u8; 32],
+    pub(crate) rsk: [u8; 32],
+    pub(crate) rvk: [u8; 32],
+    pub(crate) m: [u8; 32],
+    pub(crate) sig: [u8; 64],
+    pub(crate) rsig: [u8; 64],
+}
+
+pub(crate) fn make_test_vectors() -> Vec<TestVector> {
+    // From https://github.com/zcash/zcash-test-vectors/blob/master/zcash_test_vectors/sapling/redjubjub.py
```

Comment:
I have not checked these.

Reply:
I will check them when I review https://github.com/zcash/zcash-test-vectors/pull/95 .